pkgs/by-name/e-/e-imzo-manager/package.nix

```
@@ -0,0 +1,95 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  appstream,
+  appstream-glib,
+  cargo,
+  clippy,
+  desktop-file-utils,
+  gettext,
+  git,
+  gnome-desktop,
+  meson,
+  ninja,
+  pkg-config,
+  polkit,
+  rustc,
+  rustPlatform,
+  wrapGAppsHook4,
+  gdk-pixbuf,
+  glib,
+  adwaita-icon-theme,
+  gtk4,
+  libadwaita,
+  openssl,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "e-imzo-manager";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "xinux-org";
+    repo = "e-imzo";
+    tag = finalAttrs.version;
+    hash = "sha256-uDaqkz2VDvqTgi+k8EGGKjLkjoH93xXHQcgUc1NVo30=";
+  };
+
+  cargoDeps =
+    with finalAttrs;
+    rustPlatform.fetchCargoVendor {
+      inherit pname version src;
+      hash = "sha256-rulWG4L/uN6+JBk+SzC0y57Pdw5N0Q1dJlpXGVo+vbQ=";
+    };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    appstream
+    appstream-glib
+    cargo
+    clippy
+    desktop-file-utils
+    gettext
+    git
+    meson
+    ninja
+    pkg-config
+    polkit
+    rustc
+    rustPlatform.cargoSetupHook
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    desktop-file-utils
+    gdk-pixbuf
+    glib
+    gnome-desktop
+    adwaita-icon-theme
+    gtk4
+    libadwaita
+    openssl
+    rustPlatform.bindgenHook
+    polkit.dev
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  LD_LIBRARY_PATH = lib.makeLibraryPath [
+    polkit.dev
+  ];
```

```suggestion
    polkit
  ];

  propagatedUserEnvPkgs = [ polkit ];

  postInstall = ''
    wrapProgram $out/bin/E-IMZO-Manager \
      --prefix PATH : ${lib.makeBinPath finalAttrs.propagatedUserEnvPkgs}
  '';

  passthru.updateScript = nix-update-script { };
```

This is enough to pull in `polkit` as a runtime dependency in the system `PATH`.